spec/src/modules/search.js

```diff
@@ -6,6 +6,7 @@ const sinon = require('sinon');
 const sinonChai = require('sinon-chai');
 const ConstructorIO = require('../../../test/constructorio'); // eslint-disable-line import/extensions
 const helpers = require('../../mocha.helpers');
+const utilsHelpers = require('../../../src/utils/helpers');
```

Comment:
Feel free to ignore this, but should we just import the specific function here so we don't have helpers and utilsHelpers? Haha

i.e. ```const { trimNonBreakingSpaces } = require('../../../src/utils/helpers');```

Reply:
Great callout - this is going to go away anyway. We should not be trimming on search either